index.html

```diff
@@ -151,7 +151,7 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
         <pre class="idl">
```

Comment:
The text above should be edited as well. It currently says:

```
          CropTarget is an intentionally empty, opaque identifier that exposes nothing. Its sole
          purpose is to be handed to {{BrowserCaptureMediaStreamTrack/cropTo}} as input.
```

Reply:
The "exposes nothing" part was what triggered me as `CropTarget.fromElement()` is something to me.
Your call though ;)

Reply:
I could remove "exposes nothing". Would that make things clearer?

Reply:
Done.

Reply:
LGTM